pkgs/top-level/all-packages.nix

```
@@ -475,6 +475,8 @@ in
 
   fetchFromSavannah = callPackage ../build-support/fetchsavannah {};
 
+  fetchFromSourcehut = callPackage ../build-support/fetchsrht { };
```

What's the current status of this? From a quick look the diff LGTM. Anything left to change/improve/review?

Nitpick: Renaming the file might be more appropriate now but it doesn't really matter so feel free to ignore:
```suggestion
  fetchFromSourcehut = callPackage ../build-support/fetchsourcehut { };
```

---

> Anything left to change/improve/review?

No, I guess it's ready.

---

Awesome, thanks! :)